exercises/practice/bob/.docs/instructions.append.md

```diff
@@ -0,0 +1 @@
+Implement a function called `response` which takes a string as input and produces Bob's response as output..
```

Comment:
Punctuation again... this one ends with two periods :)

Reply:
oh, that's where I put the period missing from above